### DISCUSSION TITLE
Inquiry About Maintenance and Potential Collaboration

### DISCUSSION THREAD

**Discussion by jose-artur** [FlaUI/FlaUI#661] [General]
Hello @Roemer ,
 My team and I are exploring automation tools, and we are particularly interested in using FlaUI for our project.

We  wanted to inquire about its current maintenance status. Additionally, we also want to understand if there are opportunities for collaboration.

Would it be possible to arrange a conversation with you to discuss this further? Our team is also open to contributing to the project, potentially as maintainers.
Looking forward to your response!

**Comment by Roemer** [comment]
Hello @jose-artur 

Thanks for contacting me. As you probably figured out, maintenance is fairly minimal now. A new release will be published today or tomorrow (everything is prepared already) which removes old .NET compatibility and finally updates the dependencies which have security vulnerabilities.

In general, I am very open for additional maintainers. Most issues currently arriving are with the implemented control wrappers (like dropdown) or issues with not being able to find controls. I suppose the base framework (which is actually just a wrapper around UIA) should be very stable. If you or your team has ideas for improvements or could even assist with issues, I would gladly accept that.

**Reply by Grzegorz1223** [reply]
Hello! Are you still planning to release the new version?

**Reply by Roemer** [reply]
It is just released. Feel free to report issues.
